concepts/05 UI Components/Toast/00 Getting Started with Toast/10 Customize Toast Content.md

```diff
@@ -0,0 +1,277 @@
+                <div id="buttons">
+                    <!-- ... -->
+                    <div id="custom"></div>
+                </div>
+            </div>
+        </body>
+    </html>
+
+    <!-- tab: index.css -->
+    /*  */
+
+    .dx-custom-toast {
+        background-color: #F05B41;
+        border-radius: 20%;
+        padding: 17px;
+    }
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <div id="container">
+        <div id="buttons">
+            <!-- ... -->
+            <dx-button
+                text="Show custom message"
+                (onClick)="showCustomMessage()"
+            >
+            </dx-button>
+            <dx-toast
+                [(visible)]="isVisible"
+                [width]="230"
+                [height]="70"
+                type="custom"
+            >  
+                <dxo-position
+                    my="bottom"
+                    at="bottom"
+                    of="#container">
+                </dxo-position>
+                <div *dxTemplate="let data of 'content'" class="dx-toast-custom">
+                    <span>You have a new message <i class="dx-icon-email"></i></span>
+                </div>
+            </dx-toast>
+        </div>
+    </div>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+    import notify from 'devextreme/ui/notify';
+
+    @Component({
+        selector: 'app-root',
+        templateUrl: './app.component.html',
+        styleUrls: ['./app.component.css']
+    })
+    export class AppComponent {
+        // ...
+        isVisible: boolean = false;
+        showCustomMessage() {
+            this.isVisible = true;
+        }
+    }
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxButtonModule, DxToastModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxButtonModule,
+            DxToastModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+    <!-- tab: app.component.css -->
+    /*  */
+
+    .dx-toast-custom {
+        background-color: #F05B41;
+        border-radius: 5%;
+        padding: 2px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div id="container">
+            <div id="buttons">
+                <!-- ... -->
+                <dxButton
+                    text="Show custom message"
+                    @click="showCustomMessage"
+                >
+                </dxButton>
+            </div>
+            <dxToast
+                v-model:visible="isVisible"
+                :content-template="customMessage"
```

Comment:
why do u use this api? I think it's better to use Vue templates as a more native variant